README.md

```diff
@@ -1,20 +1,20 @@
 # cloud.gov Demos
 
-This project provides several live interactive demonstrations of 
-cloud.gov functionality.
+This project provides several a interactive demonstration of
```

Comment:
just a nitpick, either change this to: This project provides several demonstrations ...
or This project provides an interactive demonstration...